src/ManageCourses.Api/Startup.cs

```diff
@@ -52,17 +52,15 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddScoped<IManageCoursesDbContext>(provider => provider.GetService<ManageCoursesDbContext>());
 
-            services.AddAuthentication(options =>
-            {
-                options.DefaultAuthenticateScheme = BearerTokenDefaults.AuthenticationScheme;
-                options.DefaultChallengeScheme = BearerTokenDefaults.AuthenticationScheme;
-            }).AddBearerToken(options =>
-            {
-                options.UserinfoEndpoint = Configuration["auth:oidc:userinfo_endpoint"];
-            }).AddBearerTokenApiKey(options =>
-            {
-                options.ApiKey = Configuration["api:key"];
-            });
+            services.AddAuthentication()
```

Comment:
```
services.AddAuthentication(
 // And explain it here of the available choices
 // As AuthorizeAttribute was designed to use a default that we no longer set.
)
```

Reply:
Not sure what to write here, suggestions?

Reply:
use one of the following?

Reply:
BearerTokenAuthorize and its counterpart BearerTokenApiKeyAuthorize as `Authorize` is dead?

Reply:
done